test/e2e/batch_v1_jobs.go

```diff
@@ -1,342 +0,0 @@
-/*
```

Comment:
We have identical in `test/e2e/jobs.go`, originally when the two were written they were testing different endpoints (jobs.go - extensions/v1beta1 and batch_v1_jobs.go - batch/v1 Jobs), but from what I see, some time ago both were switched to Batch client and that's how it's left. I'm removing this one and leaving the other, to keep the naming scheme in-line with other resources.